json_journal/preconditioning.py

```diff
@@ -0,0 +1,49 @@
+'''Data sorted before being used by analysis'''
+
+
+def sort_journal_by_date(ledger):
+    '''
+    Sort JSON file by date.
+
+    Parameters
+    ----------
+    Ledger : dict
+        Dictionary with dates, descriptions and entries.
+
+    Returns
+    -------
+    Sorted_ledger : dict
+        Dictionary sorted by date with descriptions and entries.
+    '''
+    return sorted(ledger, key=lambda d: d['date'])
+
+
+def check_entries_sum_to_zero(ledger):
+    '''
+    Check entries in a transaction sum to zero.
+
+    Parameters
+    ----------
+    Ledger : dict
+        Dictionary with dates, descriptions and entries.
+
+    Returns
+    -------
+    Print statement
```

Comment:
I don't think this is returned, more of a side-effect.

Reply:
removed